bertinNDAYISHIMIYE/capstoneProject-MyBrand
# ft-blogTest

###What does this PR do?

>add dummy data to use in testing
>add blog test logic
>add image file used in posting blog
>add package.json modification

###Description of the task to be completed!
>Test the blog endpoints

###How should this be tested manually?

>Clone this repo with: https://github.com/bertinNDAYISHIMIYE/capstoneProject-MyBrand
>Checkout to ft-blogTest branch
>Make sure node.js is installed
>Run command npm init for npm initialisation

###Any background context you want to provide?

N/P
###Relavant trello card
https://trello.com/c/d359ANvd
